spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/orm/jpa/HibernateProperties.java

```diff
@@ -56,6 +57,13 @@
 	 */
 	private Boolean useNewIdGeneratorMappings;
 
+	/**
+	 * Archive Scanner. This is actually a shortcut for the "hibernate.archive.scanner"
+	 * property. When not specified will default to
+	 * "org.hibernate.boot.archive.scan.internal.DisabledScanner".
+	 */
+	private String archiveScanner = "org.hibernate.boot.archive.scan.internal.DisabledScanner";
```

Comment:
Thanks for the update but that's not what I meant. There shouldn't be any property at all.

Reply:
property was removed